src/Slices/ServiceInstanceDetails/UI/Tabs/DocumentationTabContent.tsx

```diff
@@ -49,6 +49,14 @@ export const DocumentationTabContent: React.FC<Props> = ({
   const { logsQuery } = useContext(InstanceDetailsContext);
   const [expanded, setExpanded] = useState(0);
 
+  if (logsQuery.isLoading) {
+    return (
+      <TabContentWrapper id={"documentation"}>
+        <LoadingView />
+      </TabContentWrapper>
+    );
+  }
+
   if (!logsQuery.data) {
```

Comment:
here as well, we don't handle error if I read through file correctly, but maybe that's fine

Reply:
it is handled in a higher level, so I didn't add it here, only for the case we don't have data but the query succeeded somehow. I'll go over the error handling when I have all the views, but each section can potentially go wrong in some way, so it will have some redundant errorhandling here and there